scielomanager/journalmanager/models.py

```diff
@@ -523,6 +523,8 @@ class Journal(models.Model):
     use_license = models.ForeignKey('UseLicense', verbose_name=_('Use license'),
             default=get_journals_default_use_license)
     collections = models.ManyToManyField('Collection', through='Membership')
+
+    jid = models.CharField(max_length=32, unique=True, editable=False)
```

Comment:
Acho que teria que ter o mesmo tamanho do campo short_title caso o short_title for a fonte de produção do jid